jenkins/jobs/image-pld.yaml

```diff
@@ -1,5 +1,3 @@
-- job:
-    name: "image-pld"
```

Comment:
What happened here?

Reply:
Whoa, probably bogus keystrokes in vim, fixed now.